### PR TITLE
Add Documentation Link and Supported Versions

### PR DESCRIPTION
**Pull request by CrshOverride**
Addresses #7168.

- Adds a link to the `node-support.md` document in the console messages if there's a Node version issue.
- Updates `node-support.md` with version ranges for past and current Node JS releases.

### CODE REVIEW

File: lib/cli/cli.js [hunk 1]
```diff
@@ -120,7 +120,8 @@ class CLI {
 
       const PlatformChecker = require('../utilities/platform-checker');
       let platform = new PlatformChecker(process.version);
-      let recommendation = ' We recommend that you use the most-recent "Active LTS" version of Node.js.';
+      let recommendation = ' We recommend that you use the most-recent "Active LTS" version of Node.js.' +
+                           ' See https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md for details.';
```
Comment (rwjblue) [inline lib/cli/cli.js]: Thoughts on using git.io to shorten this to https://git.io/v7S5n?

**Comment by CrshOverride** [comment]
@rwjblue I much prefer the shortened link. I've made the update and pushed. Any idea why tests would be failing because of the one-line Markdown change in `2ae045c`?

**Comment by rwjblue** [comment]
The specific failures make no sense to me 🤔

**Comment by CrshOverride** [comment]
Looks like a flaky test or two. Changing the link "fixed" it.

**Comment by rwjblue** [comment]
Thank you!!
